client/src/components/feed.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import './feed.css';
+import Feed from './feedComponent';
+import profileImg from './imgAvatar.png';
+
+function FeedPage() {
+
+    //dummy data
+    var feedData = [
```

Comment:
It would be great if you create another dummy data file and fetch from there.
Reason:- Code becomes clean and easy to fetch backend as well.

Reply:
how can this be achieved?
By adding a .txt file or something?
Can you plz provide some references for the same, where I can learn about it?

Reply:
Not a .txt , I will give you a better idea.

create one js file call `dummydata.js` and paste `feedData` by the object also export from this file.

after that using the import/require statement take a data object and then use it.

if isn't work let me know.

Reply:
Ok, thanks for the help @codewithvk!